src/parser/mumexmlparser.cpp

```diff
@@ -338,8 +339,13 @@ QByteArray MumeXmlParser::characters(QByteArray& ch)
       m_lastPrompt = ch;
   }
 
-  m_stringBuffer = ch.simplified();
-  latinToAscii(m_stringBuffer);
+  if (Config().m_utf8Charset)
+      m_stringBuffer = QString::fromUtf8(ch);
```

Comment:
Do we need to convert this to Latin-1? We don't have an equivalent `ParserUtils::utf8ToAscii()`